pkger/chronograf.go

```diff
@@ -0,0 +1,17 @@
+package pkger
+
+import "github.com/influxdata/influxdb"
+
+// DashboardToResource is an exported call to dashboardToResource used by the
+// chronograf-migrator. It is in a separate file so that it may be maintained distinctly
+// from the internal one.
+func DashboardToResource(d influxdb.Dashboard, name string) Resource {
```

Comment:
can these just be added to the `clone_resource.go` file intead? Feel free to export the functions that this are called here. Don't see any harm in it 👍

Reply:
Yeah, they totally can. I moved them to a separate file so that I didn't keep hitting merge conflicts during rebases.

Reply:
smart mvoe